documentation/config_logging_advanced.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "Puppet Server: Advanced Logging Configuration"
+canonical: "/puppetserver/latest/config_logging_advanced.html"
+---
+
+Puppet Server uses the very flexible, very powerful [Logback](http://logback.qos.ch/) library under the hood to handle all of our logging needs.
+
+Among the many interesting and useful capabilities of logback is the ability to configure it to log messages in a JSON format, which makes it easy to send them over to other logging backends such as logstash.
+
+## Configuring Puppet Server for use with Logstash
+
+There are a few steps necessary to get your Puppet Server logging set up for use with logstash.  The first step is to modify your logging configuration so that Puppet Server is logging in a JSON format.  After that, you'll need to configure an external tool to monitor these JSON files and send the data to logstash (or another remote logging system).
+
+### Configuring Puppet Server to log to JSON
+
+There are a handful of considerations you may want to take into account before configuring Puppet Server to log to JSON:
+
+* Do you want to configure Puppet Server to *only* log to JSON?  In lieu of the normal text file logging?  Or do you want to add JSON logging *in addition to* your normal text logging?
```

Comment:
What does "normal" mean? If it means plain text, can we say that instead of "normal"? For instance:

> * Do you want to configure Puppet Server to *only* log to JSON in lieu of the default plain-text file logging? Or do you want to add JSON logging *in addition to* the default plain-text logging?